assets/js/components/notifications/GA4AdSenseLinkedNotification.test.js

```diff
@@ -19,136 +19,154 @@
 /**
  * Internal dependencies
  */
-import GA4AdSenseLinkedNotification from './GA4AdSenseLinkedNotification';
 import {
 	render,
 	createTestRegistry,
 	provideModules,
-	muteFetch,
+	provideNotifications,
+	act,
+	waitForDefaultTimeouts,
 } from '../../../../tests/js/test-utils';
+import { provideAnalytics4MockReport } from '../../modules/analytics-4/utils/data-mock';
 import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
 import { MODULES_ANALYTICS_4 } from '../../modules/analytics-4/datastore/constants';
 import {
 	VIEW_CONTEXT_MAIN_DASHBOARD,
 	VIEW_CONTEXT_SETTINGS,
 } from '../../googlesitekit/constants';
-import { withNotificationComponentProps } from '../../googlesitekit/notifications/util/component-props';
+import { DEFAULT_NOTIFICATIONS } from '../../googlesitekit/notifications/register-defaults';
+import { NOTIFICATION_AREAS } from '../../googlesitekit/notifications/datastore/constants';
+import Notifications from './Notifications';
 
 const GA4_ADSENSE_LINKED_NOTIFICATION =
 	'top-earning-pages-success-notification';
 
 // eslint-disable-next-line jest/no-disabled-tests
```

Comment:
```suggestion
```

Reply:
Oopsie - thanks! That was the point of the issue! 😄